journey/README.md

```diff
@@ -107,12 +107,39 @@ val journey = Journey {
     module(Oidc) {
         clientId = "your_client_id"
         discoveryEndpoint =
-                "https://your_openam_domain/am/oauth2/alpha/.well-known/openid-configuration"
+            "https://your_openam_domain/am/oauth2/alpha/.well-known/openid-configuration"
         // ... other OIDC configurations
+
+        // Storage configuration options
+        storage {
+            fileName = "test"
+            keyAlias = "myKeyAlias"
+            strongBoxPreferred = true
+            cacheStrategy = CacheStrategy.CACHE_ON_FAILURE
+        }
+    }
+    module(Session) {
+        storage {
+            //keep default filename and keyalias.
+            strongBoxPreferred = false
+            cacheStrategy = CacheStrategy.CACHE_ON_FAILURE
+        }
     }
 }
 ```
 
+#### Storage Configuration Explained
+
+The `storage` block allows you to configure how data is persisted for each module. The options include:
+
+- **fileName**: The name of the file used for persistent storage.
+- **keyAlias**: If provided, enables encryption using AndroidKeyStore. This results in the use of `EncryptedDataStoreStorage`.
+- **strongBoxPreferred**: If set to `true`, attempts to use hardware-backed StrongBox for key storage (if available).
+- **cacheStrategy**: Controls in-memory caching behavior. Options:
```

Comment:
Call out the risk of using cache strategy here as well.

Reply:
Updated